demo_notebook/visualisations demo notebook.ipynb

```diff
@@ -0,0 +1,90 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# we need folium for the notebook\n",
+    "!pip install folium==0.12.1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from openspoor.visualisations.spoorkaart import SpoorKaart,PlottingDataFrame"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Set up spoorkaart and add points to plot\n",
+    "\n",
+    "\n",
+    "Making a map consists of setting up a SpoorKaart object and adding the objects you wish to plot to it. These objects can be given in Pandas DataFrames, which are displayed on a map with added aerial photographs of the Dutch tracks and zoomed to the location of interest.\n",
+    "Optionally, these outputs can be saved as .html files which can then be shared or used in applications.\n",
+    "Within notebooks, Spoorkaart objects are displayed if their value is requested at the end of a cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with SpoorKaart() as m:\n",
+    "    PlottingDataFrame({'lat': [52.08, 52.093],\n",
```

Comment:
Misschien een comment per plot die je uitvoert te verduidelijking

Reply:
Deze is verplaatst naar het andere notebook met extra inline comments